docs/tutorial/quickstart.md

```diff
@@ -0,0 +1,355 @@
+# Tutorial: Get started with oras-go v2
+
+This tutorial introduces the basics of managing OCI artifacts with the [oras-go v2](https://pkg.go.dev/oras.land/oras-go/v2) package.
+
+You'll get the most out of this tutorial if you have a basic familiarity with Go and its tooling. If this is your first exposure to Go, please see [Tutorial: Get started with Go](https://golang.org/doc/tutorial/getting-started) for a quick introduction.
+
+The tutorial includes the following sections:
+
+1. Create a folder for your code.
+2. Connect to a remote repository.
+3. Show tags in the repository.
+4. Push a layer to the repository.
+5. Push a manifest to the repository.
+6. Fetch the manifest from the repository.
+7. Parse the fetched manifest content and get the layers.
+8. Copy the artifact from the repository.
+
+The complete code is provided at the end of this tutorial.
+
+## Create a folder for your code
+
+Open a command prompt and cd to your home directory.
+
+On Linux or Mac:
```

Comment:
```suggestion
On Linux or macOS:
```

Reply:
Replaced this part with "Open a command prompt and cd to your working directory."